hybris/bin/custom/tatacliq/tatacliqstorefront/web/src/de/hybris/tatacliq/storefront/controllers/cms/CustomComponentController.java

```diff
@@ -0,0 +1,40 @@
+package de.hybris.tatacliq.storefront.controllers.cms;
+
+import de.hybris.platform.acceleratorstorefrontcommons.controllers.cms.AbstractCMSComponentController;
+import de.hybris.tatacliq.core.model.CustomComponentModel;
+import de.hybris.tatacliq.storefront.controllers.ControllerConstants;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+
+@Controller("CustomComponentController")
+@Scope("tenant")
+@RequestMapping(value = ControllerConstants.Actions.Cms.CustomComponent)
+public class CustomComponentController extends AbstractCMSComponentController<CustomComponentModel>
```

Comment:
Rename this to AirpointsComponentController

Reply:
you first check if there is still any conflicts.. After that I will change the name of the files.